spec/lib/nacre/order_collection_spec.rb

```diff
@@ -42,7 +42,11 @@
     let(:get_options) do
       {
         search_url: order_service_url,
-        ids: range
+        ids: range,
+        options: [
+          'customFields',
+          'nullCustomFields'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.